### PR TITLE
doc: update dev env vagrant instructions

### PR DESCRIPTION
**Pull request by theanalyst**
updating dev env Vagrant run instuctions to provide link on running server with supervisord, since otherwise this can be found only on the youtube video

Also fixed a minor typo s/build/built

Signed-Off-By: Abhishek Lekshmanan <abhishek.lekshmanan@ril.com>

### CODE REVIEW

File: doc/development/dev_env.rst [hunk 1]
```diff
@@ -73,7 +74,7 @@ For RH systems:
 
 
 4. Grab the `GUI code <https://github.com/ceph/calamari-clients>`_, build it and
-   place the build products in ``webapp/content`` so that when it's installed you
+   place the built products in ``webapp/content`` so that when it's installed you
```
Comment (dmick) [inline doc/development/dev_env.rst]: Not an error, but either word works.

File: doc/development/dev_env.rst [hunk 2]
```diff
@@ -201,4 +202,4 @@ Authorize the simulated salt minions to connect to the calamari server:
     salt-key -c dev/etc/salt -A
 
 You should see some debug logging in cthulhu.log, and if you visit /api/v2/cluster in your browser
-a Ceph cluster should appear.
\ No newline at end of file
+a Ceph cluster should appear.
```
Comment (dmick) [inline doc/development/dev_env.rst]: What was the change here?

File: doc/development/dev_env.rst (hunk 1 above)
Comment (theanalyst) [inline doc/development/dev_env.rst]: ah ok, in which case I'm ok to roll back

File: doc/development/dev_env.rst (hunk 2 above)
Comment (theanalyst) [inline doc/development/dev_env.rst]: oops sorry, I think it was an editor setting that sets a newline at end of file.. I'll roll this back as well

Comment (dmick) [inline doc/development/dev_env.rst]: weird; the diff shows it well in this view, but did not in others.  makes sense.

Comment (theanalyst) [inline doc/development/dev_env.rst]: ah ok.. do you want me to change anything then for this patch

Comment (dmick) [inline doc/development/dev_env.rst]: It'd be better to avoid the gratuitous change if we can.

Comment (theanalyst) [inline doc/development/dev_env.rst]: sure.. revert the newline change

**Pull request merged**